miio/integrations/vacuum/roborock/vacuumcontainers.py

```diff
@@ -656,6 +656,32 @@ def dustbin_auto_empty_used(self) -> Optional[int]:
             return self.data["dust_collection_work_times"]
         return None
 
+    @property
+    @sensor(
+        "Strainer used",
+        icon="mdi:air-filter",
+        entity_category="diagnostic",
+        enabled_default=False,
+    )
+    def strainer_used(self) -> Optional[int]:
+        """Return ``strainer_work_times``"""
+        if "strainer_work_times" in self.data:
+            return self.data["strainer_work_times"]
+        return None
+
+    @property
+    @sensor(
+        "Cleaning brush used",
+        icon="mdi:brush",
+        entity_category="diagnostic",
+        enabled_default=False,
+    )
+    def cleaning_brush_used(self) -> Optional[int]:
+        """Return ``cleaning_brush_work_times``"""
```

Comment:
Same for this, is this also a count? Please use descriptive docstrings to describe what type of information is returned instead of just naming what is the internal name for the piece of information.